test/test_art_scan.cpp

```
@@ -0,0 +1,999 @@
+// Since the data only contains the ODD keys, you can probe with EVEN
+// keys and verify that the scan() is carried out from the appropriate
+// key in the data when the from_key and/or to_key do not exist in the
+// data.
+//
+// @param from_key
+//
+// @param to_key
+//
+// @param limit The largest key to be installed (ODD).
+template <typename TypeParam>
+void do_scan_range_test(const unodb::key from_key, const unodb::key to_key,
```

cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If UNODB_DETAIL_DISABLE_CLANG_WARNING is a macro then please configure it.

(unknownMacro)